docs/installation.md

```diff
@@ -241,22 +226,21 @@ docker-compose up
 
 Watching the log messages, you will see that after some time, the elections process completes. Then you're able to connect to each
 node using the Admin UI. Nodes should be accessible on the loopback address (`127.0.0.1` or `localhost`) over
-HTTP and TCP, using ports specified below:
+HTTP, using ports specified below:
 
-| Node  | TCP port | HTTP port |
-| :---- | :------- | :-------- |
-| node1 | 1111     | 2111      |
-| node2 | 1112     | 2112      |
-| node3 | 1113     | 2113      |
+| Node  | HTTP port |
+| :---- | :-------- |
+| node1 | 2111      |
+| node2 | 2112      |
+| node3 | 2113      |
 
-You have to tell your client to use secure connection for both TCP and gRPC.
+You have to tell your client to use secure connection.
 
 | Protocol | Connection string                                                                                     |
 | :------- | :---------------------------------------------------------------------------------------------------- |
-| TCP      | `GossipSeeds=localhost:1111,localhost:1112,localhost:1113;ValidateServer=False;UseSslConnection=True` |
 | gRPC     | `esdb://localhost:2111,localhost:2112,localhost:2113?tls=true&tlsVerifyCert=false`                    |
 
-As you might've noticed, both connection strings have a setting to disable the certificate validation (`ValidateServer=False` for `TCP` and `tlsVerifyCert=false` for `gRPC`). It would prevent the invalid certificate error since the cluster uses a private, auto-generated CA.
+As you might've noticed, the connection string has a setting to disable the certificate validation (`ValidateServer=False` for `TCP` and `tlsVerifyCert=false` for `gRPC`). It would prevent the invalid certificate error since the cluster uses a private, auto-generated CA.
```

Comment:
```suggestion
As you might've noticed, the connection string has a setting to disable the certificate validation (`tlsVerifyCert=false`). It would prevent the invalid certificate error since the cluster uses a private, auto-generated CA.
```